src/lib/registration/registration.rb

```diff
@@ -164,6 +165,25 @@ def migration_products(installed_products)
       migrations
     end
 
+    # Get the list of updates for a given product
+    #
+    # @param [Hash]           Product. If nil the base product will be used.
```

Comment:
I think this really need description how hash should look like. Just by reading documentation I have no clue what to pass.